internal/service/enrollmentrequest.go

```diff
@@ -19,13 +19,13 @@ import (
 const ClientCertExpiryDays = 365
 
 type EnrollmentRequestStore interface {
-	CreateEnrollmentRequest(ctx context.Context, orgId uuid.UUID, req *api.EnrollmentRequest) (*api.EnrollmentRequest, error)
-	ListEnrollmentRequests(ctx context.Context, orgId uuid.UUID, listParams ListParams) (*api.EnrollmentRequestList, error)
-	GetEnrollmentRequest(ctx context.Context, orgId uuid.UUID, name string) (*api.EnrollmentRequest, error)
-	CreateOrUpdateEnrollmentRequest(ctx context.Context, orgId uuid.UUID, enrollmentrequest *api.EnrollmentRequest) (*api.EnrollmentRequest, bool, error)
-	UpdateEnrollmentRequestStatus(ctx context.Context, orgId uuid.UUID, enrollmentrequest *api.EnrollmentRequest) (*api.EnrollmentRequest, error)
-	DeleteEnrollmentRequests(ctx context.Context, orgId uuid.UUID) error
-	DeleteEnrollmentRequest(ctx context.Context, orgId uuid.UUID, name string) error
+	Create(ctx context.Context, orgId uuid.UUID, req *api.EnrollmentRequest) (*api.EnrollmentRequest, error)
+	List(ctx context.Context, orgId uuid.UUID, listParams ListParams) (*api.EnrollmentRequestList, error)
+	Get(ctx context.Context, orgId uuid.UUID, name string) (*api.EnrollmentRequest, error)
+	CreateOrUpdate(ctx context.Context, orgId uuid.UUID, enrollmentrequest *api.EnrollmentRequest) (*api.EnrollmentRequest, bool, error)
+	UpdateStatus(ctx context.Context, orgId uuid.UUID, enrollmentrequest *api.EnrollmentRequest) (*api.EnrollmentRequest, error)
+	DeleteAll(ctx context.Context, orgId uuid.UUID) error
```

Comment:
Still not sure DeleteAll should be a dependency of the interface vs an implementation detail